tools/extra/hssi/ethernet/hssicommon.py

```diff
@@ -656,6 +656,14 @@ def eth_mode(self):
         return self.bits.eth_mode
 
 
+def veriy_pcie_address(pcie_address):
+        m = BDF_PATTERN.match(pcie_address)
+        if m is None:
+            print("Invalid pcie address foramt",pcie_address)
+            return False
+        return True
```

Comment:
I think the indentation is too deep.  Use 4 spaces instead.

Reply:
fixed